beta/src/content/learn/importing-and-exporting-components.md

```diff
@@ -1,26 +1,26 @@
 ---
-title: Importing and Exporting Components
+title: 導入及導出 Component
 ---
 
 <Intro>
 
-The magic of components lies in their reusability: you can create components that are composed of other components. But as you nest more and more components, it often makes sense to start splitting them into different files. This lets you keep your files easy to scan and reuse components in more places.
+Component 的神奇之處在於它的可複用性： 你可以創建一個由其他 component 組成的 component。但當你嵌套越來越多 component，則需要開始將它們拆分為不同的檔案。這將會提升檔案的閱讀性，也能讓 component 重複應用在更多地方。
 
 </Intro>
 
 <YouWillLearn>
 
-* What a root component file is
-* How to import and export a component
-* When to use default and named imports and exports
-* How to import and export multiple components from one file
-* How to split components into multiple files
+* 什麼是根 component 檔案
+* 如何導入以及導出一個 component
+* 何時使用預設和具名導入導出
+* 如何從一個檔案導入以及導出多個 component
+* 如何將 component 拆分為多個檔案
 
 </YouWillLearn>
 
-## The root component file {/*the-root-component-file*/}
+## 根 component 檔案 {/*the-root-component-file*/}
 
-In [Your First Component](/learn/your-first-component), you made a `Profile` component and a `Gallery` component that renders it:
+在 [你的第一個 Component](/learn/your-first-component) 中，你創建了一個 `Profile` component，並且 render 在 `Gallery` component 裡：
```

Comment:
```suggestion
在 [你的第一個 Component](/learn/your-first-component) 中，你創建了一個 `Profile` component，並且呈現在 `Gallery` component 裡：
```

`render` => `算繪`、`呈現`

Reply:
I think we can keep the `render` instead of translate it to Mardian.

```suggestion
在 [你的第一個 Component](/learn/your-first-component) 中，你建立了一個 `Profile` component，並且 render 在 `Gallery` component 裡：
```